spec/posts_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] You should have  `posts_spec.rb` in `spec/requests/` directory with `type: request`